### ISSUE TITLE
Scaffolding fails when used against a project that uses the new minimal hosting APIs in ASP.NET Core 6

### ISSUE THREAD

**Issue by DamianEdwards** [dotnet/Scaffolding#1584]
ASP.NET Core introduces new minimal APIs for bootstrapping the application `WebApplication.Create` and `WebApplication.CreateBuilder`. These APIs are used instead of `Host.CreateBuilder` and do not involve the declaration and use of a `Startup` class. So far only the empty web app template in .NET 6 uses these APIs.

Here's an example `Program` from a project using these APIs:
``` csharp
using System;
using Microsoft.AspNetCore.Builder;
using Microsoft.Extensions.Hosting;

namespace WebApplication55
{
    public class Program
    {
        public static void Main(string[] args)
        {
            var builder = WebApplication.CreateBuilder(args);
            var app = builder.Build();

            if (app.Environment.IsDevelopment())
            {
                app.UseDeveloperExceptionPage();
            }

            app.MapGet("/", (Func<string>)(() => "Hello World!"));

            app.Run();
        }
    }
}
```

Attempting to add a scaffolded item (e.g. API controller using Entity Framework Core) results in an error, e.g. in Visual Studio:

```
---------------------------
Microsoft Visual Studio
---------------------------
Error

There was an error running the selected code generator:

'Scaffolding failed to edit Startup class to register the new Context using Dependency Injection. Make sure there is a Startup class and a ConfigureServices method and Configuration property in it.'
---------------------------
OK   
---------------------------
```

**Comment by deepchoudhery** [comment]
Oh I see. I'll get started on the next previews for .NET 6 pretty soon. I'll play with the new .NET 6 preview bits and see what needs to be done. I definitely won't be able to get this done for .NET 6 Preview 5 but definitely for .NET Preview 6.
I'll have updates here.

**Comment by DamianEdwards** [comment]
@deepchoudhery any update on this?

**Comment by markrendle** [comment]
Is this going to be fixed in RC1? Or is it not going to be supported at all in the future?

With .NET 6 Preview 7:

```
FsxIdentity/src/FsxIdentity via .NET v6.0.100-preview.7.21379.14 🎯 net6.0
❯ dotnet aspnet-codegenerator identity -dc FsxIdentity.Data.ApplicationDbContext
Building project ...
Finding the generator 'identity'...
Running the generator 'identity'...

Scaffolding Identity is not supported for web applications with minimal APIs for hosting and routing.

RunTime 00:00:13.32
```

cc @deepchoudhery @DamianEdwards

**Comment by deepchoudhery** [comment]
@markrendle Hey, will be fixed for rc1. Already working in internal packages.

**Comment by vijayrkn** [comment]
This is already fixed in 6.0 rc packages. Closing.
